src/FantomasVs.Shared/FantomasVsPackage.cs

```diff
@@ -67,6 +68,17 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
 
         }
 
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                Trace.WriteLine("Fantomas Vs Package Disposing");
+                FantomasService.Dispose();
```

Comment:
Will this ever throw?
Should we add a `try-catch`?

Reply:
It will throw by its own code but might throw when someone tampered with the spun up Fantomas processes.